configs/bb.org-debian-stretch-iot-grove-kit-v4.14.conf

```diff
@@ -148,6 +148,7 @@ repo_rcnee_pkg_list="	\
 	firmware-am57xx-opencl-monitor	\
 	gpiod	\
 	linux-image-4.14.108-ti-r127	\
+	seeed-modules-4.14.108-ti-r127	\
```

Comment:
This is package is already installed:

https://github.com/beagleboard/image-builder/blob/master/scripts/chroot.sh#L730-L746

Regards,

Reply:
Strangely, we did not find it in the Image.
 https://rcn-ee.net/rootfs/bb.org/testing/2020-01-23/stretch-iot-music-kit/

Reply:
I just looked at the build log:

```E: Invalid operation seeed-modules-4.14.108-ti-r126
Reading package lists...
Building dependency tree...
Reading state information...
The following NEW packages will be installed:
  ti-cmem-4.15.00.02-modules-4.14.108-ti-r126
```
humm...

I'm an idiot sometimes.. (it helps if apt-get has the "install" keyword on it..)

https://github.com/beagleboard/image-builder/commit/954106b65ee786f4e7d3a9605f41deea458e6620

Regards,